docs/api/sources.md

```diff
@@ -161,6 +161,33 @@ pipe(
 );
 ```
 
+## fromPromise
+
+`fromPromise` transforms a promise into a source, emitting the promisified value on
+the source once it resolves.
+
+> _Note:_ This source is only available in JavaScript environments, and will be excluded
+> when compiling natively.
+
+```reason
+let promise = Js.Promise.resolve(1);
```

Comment:
It might be nicer here to show `Promise` creation to really highlight how `fromPromise` works. Something like:

```reason
let promise = Js.Promise.make((~resolve, ~reject as _) => resolve(. 1));
Wonka.fromPromise(promise) |> Wonka.subscribe((. x) => Js.log(x));
```

Reply:
I don't really want the example promise to be the main focus so I chose to just use `Promise.resolve` (same for the TS example). I'll just add a comment to that line and that should do the trick I think